src/test/rootElementChangeDoesNotUnmount/index.ts

```diff
@@ -0,0 +1,32 @@
+import { JSDOM } from "jsdom";
```

Comment:
Do we want new tests to use the structure [proposed in #44](https://github.com/forgojs/forgo/pull/44/files#diff-791330a3cf5be13952e2f23cb5092a5702fd38d379c03cfe0727588e08a30e43R1)?

Reply:
Ah yes, will use.